R/projpred.R

```diff
@@ -139,26 +164,44 @@ get_refmodel.brmsfit <- function(object, newdata = NULL, resp = NULL,
     # }
   }
   
+  if (utils::packageVersion("projpred") <= "2.0.2") {
```

Comment:
I think we can add `&& NROW(object$ranef)` following your questions to make this check specific to multilevel models.

Reply:
Great, thanks. I'll add this.